lib/organisations_fetcher.rb

```diff
@@ -50,12 +51,27 @@ def update_ancestry(organisation_relationships)
     organisation_relationships.each do |organisation_slug, child_organisation_slugs|
       parent = Organisation.find_by!(slug: organisation_slug)
       Organisation.where(slug: child_organisation_slugs).find_each do |child_organisation|
-        # TODO: this ignores that organisations can have multiple parents. I think organisations will
-        # end up with the parent that appears last in the API response(s).
-        #
-        # Transition app implements this correctly.
+        # TODO: This ignores that organisations can have multiple parents. Instead the
+        # chosen parent will be the last parent in Whitehall, ordered alphabetically.
         child_organisation.update!(parent: parent)
       end
     end
   end
+
+  MANUAL_PARENT_FIXES = {
+    "animal-and-plant-health-agency" => "department-for-environment-food-rural-affairs",
+  }.freeze
+
+  def fix_parents_manually
+    # Signon doesn't support multiple parents. Most of the time this is fine, but for
+    # certain organisations this leads to a poor user experience and frequent support
+    # tickets.
+    MANUAL_PARENT_FIXES.each do |child_slug, parent_slug|
+      child = Organisation.find_by(slug: child_slug)
+      parent = Organisation.find_by(slug: parent_slug)
```

Comment:
These should both be `find_by!`, I think?

Reply:
I left it as `find_by` intentionally so that if an organisation was deleted in the future, this task wouldn't break, it would just ignore the organisations that no longer exist.

Reply:
Ah yes - makes sense.